auth/credentials/impersonate/example_test.go

```diff
@@ -33,10 +36,8 @@ func ExampleNewCredentials_serviceAccount() {
 		log.Fatal(err)
 	}
 
-	// TODO(codyoss): link to option once it exists.
-
-	// Use this Credentials with a client library
-	_ = creds
+	// Use this Credentials with a client library, e.g. storage
+	_, _ = storage.NewClient(context.TODO(), option.WithAuthCredentials(creds))
```

Comment:
Yeah this todo could be resolved now. I think the important part about solving it is likely describing what API to use, like in your example, in a comment. We don't want to have this module take on those two dependencies because of an example.